jib-core/src/main/java/com/google/cloud/tools/jib/api/Ports.java

```diff
@@ -45,7 +46,7 @@
    * @return the ports as a list of {@link Port}
    * @throws NumberFormatException if any of the ports are in an invalid format or out of range
    */
-  public static ImmutableSet<Port> parse(List<String> ports) throws NumberFormatException {
+  public static Set<Port> parse(List<String> ports) throws NumberFormatException {
     ImmutableSet.Builder<Port> result = new ImmutableSet.Builder<>();
```

Comment:
I think this should return a mutable set as an API.

Reply:
Hrmm... I guess this isn't really a value class. Makes sense, I'll change the underlying implementation.